docs/api/materials/ShaderMaterial.html

```diff
@@ -44,6 +44,26 @@ <h1>[name]</h1>
 				If you don't want the [page:WebGLProgram] to add anything to your shader code, you can use
 				[page:RawShaderMaterial] instead of this class.
 			</li>
+			<li>
+				You can use the directive #pragma unroll_loop in order to unroll a *for* loop in GLSL by the shader preprocessor.
+				The directive has to be placed right above the loop. The loop itself has to correspond to a defined standard.
+				<ul>
+					<li>
+						The loop has to be [link:https://en.wikipedia.org/wiki/Normalized_loop normalized/well-behaved].
```

Comment:
If "well-behaved" means normalized, then you don't have to repeat it. Just say "normalized".

>The loop itself has to correspond to a defined standard.

The loop formatting has to correspond to a defined standard.

Reply:
👍